src/components/Checkbox/Checkbox.js

```diff
@@ -26,6 +26,13 @@ export default class Checkbox extends React.Component {
       }
     };
 
+    const info = this.props.info ?
+    <div className="chi-label__help">
```

Comment:
Is this indentation correct? The linter might throw errors when trying to push to Dev / Prod

Reply:
fixed